synapse/handlers/room.py

```diff
@@ -1872,6 +1872,6 @@ async def shutdown_room(
         return {
             "kicked_users": kicked_users,
             "failed_to_kick_users": failed_to_kick_users,
-            "local_aliases": aliases_for_room,
+            "local_aliases": list(aliases_for_room),
             "new_room_id": new_room_id,
         }
```

Comment:
Having to take a copy here is really unfortunate. We need something JSON-serializable here and the new `Sequence[...]` type hint doesn't guarantee that.

Reply:
Agreed, though the cost of the copy should be low here.